doc/tutorials/acrn_configuration_tool.rst

```diff
@@ -143,7 +151,7 @@ Additional scenario XML elements:
   The start physical address in host for the VM.
 
 ``size`` (a child node of ``memory``):
-  The memory size in Bytes for the VM.
+  The memory size inbBytes for the VM.
```

Comment:
Did you really mean to make this change?

Reply:
oops. A mistake. Nice catch. I will update it. Thanks.